extensions/SampleExtension/SampleExtension.csproj

```diff
@@ -23,18 +59,33 @@
         <Content Include="ProviderAssets\**\*.png" />
         <Content Include="Templates\**\*.json" />
     </ItemGroup>
+  
+  <ItemGroup>
+    <None Remove="Package-Dev.appxmanifest" />
+  </ItemGroup>
+  
+  <ItemGroup>
+    <AppxManifest Include="Package-Dev.appxmanifest">
+      <SubType>Designer</SubType>
+    </AppxManifest>
+  </ItemGroup>
 
     <ItemGroup>
+      <PackageReference Include="Microsoft.Extensions.Hosting" Version="8.0.0" />
+      <PackageReference Include="Microsoft.Windows.CsWin32" Version="0.2.206-beta" />
       <PackageReference Include="Microsoft.Windows.CsWinRT" Version="2.0.4" />
-      <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.4.231115000" />
+      <PackageReference Include="Microsoft.Windows.DevHome.SDK" Version="0.700.544" />
       <PackageReference Include="Microsoft.Windows.SDK.BuildTools" Version="10.0.22621.2428" />
+      <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.5.240227000" />
+      <PackageReference Include="Serilog" Version="3.1.1" />
+      <PackageReference Include="Serilog.Extensions.Logging" Version="8.0.0" />
+      <PackageReference Include="Serilog.Settings.Configuration" Version="8.0.0" />
+      <PackageReference Include="Serilog.Sinks.Console" Version="5.0.1" />
+      <PackageReference Include="Serilog.Sinks.Debug" Version="2.0.0" />
+      <PackageReference Include="Serilog.Sinks.File" Version="5.0.0" />
+      <PackageReference Include="System.Text.Json" Version="8.0.0" />
```

Comment:
Are you sure you need this one? We don't include it anywhere else, it mostly seems to come in transitively. (Also, I think 8.0.4 is the version without the security flaw)

Reply:
Unneeded and removed, ty!